packages/uui-file-dropzone/lib/uui-file-dropzone.element.ts

```diff
@@ -127,36 +126,23 @@ export class UUIFileDropzoneElement extends LabelMixin('', LitElement) {
           acceptList.push(item.trim().toLowerCase());
         }
       });
+    }
 
-      while (queue.length > 0) {
-        const entry: FileSystemFileEntry = queue.shift()!;
-        if (
-          entry.isFile &&
-          (await this._isAccepted(acceptList, wildcards, entry))
-        ) {
-          fileEntries.push(entry);
-        } else if (entry.isDirectory) {
-          fileEntries.push(entry);
-          queue.push(
-            ...(await this._readAllDirectoryEntries(
-              (entry as any).createReader()
-            ))
-          );
-        }
-      }
-    } else {
-      while (queue.length > 0) {
-        const entry: FileSystemFileEntry = queue.shift()!;
-        if (entry.isFile) {
-          fileEntries.push(entry);
-        } else if (entry.isDirectory) {
-          fileEntries.push(entry);
-          queue.push(
-            ...(await this._readAllDirectoryEntries(
-              (entry as any).createReader()
-            ))
-          );
+    while (queue.length > 0) {
+      const entry = queue.shift()!;
+
+      if (entry.kind === 'file') {
+        const file = entry.getAsFile();
+        if (!file) continue;
+        if (this._isAccepted(acceptList, wildcards, file)) {
```

Comment:
I have added an await here, as method is async.

Reply:
Oh, no... I need to getsome coffee, it's not async anymore. all good.

Reply:
Haha, it happens :D Oddly enough I tested this thing before making a PR 😇